Kitodo-DataEditor/src/test/resources/testmeta.xml

```diff
@@ -0,0 +1,105 @@
+                <mods:mods xmlns:mods="http://www.loc.gov/mods/v3">
+                    <mods:extension>
+                        <kitodo:kitodo xmlns:kitodo="http://meta.kitodo.org/v1/" version="1.0">
+                            <kitodo:metadata name="TitleDocMain">Otsar ha-kavod</kitodo:metadata>
+                            <kitodo:metadata name="PublisherName">Test Publisher</kitodo:metadata>
+                            <kitodo:metadata name="TypeOfResource" type="group">
+                                <kitodo:metadata name="typDisplayForm">Handschrift</kitodo:metadata>
+                                <kitodo:metadata name="typIdentifierPPN">10457187X</kitodo:metadata>
+                                <kitodo:metadata name="typIdentifierURI">http://d-nb.info/gnd/4023287-6</kitodo:metadata>
+                            </kitodo:metadata>
+                            <kitodo:metadata type="group" name="FormerOwner">
+                                <kitodo:metadata name="authorityURI">http://d-nb.info/gnd/</kitodo:metadata>
+                                <kitodo:metadata name="authorityValue">http://d-nb.info/gnd/</kitodo:metadata>
+                                <kitodo:metadata name="displayName">, </kitodo:metadata>
+                            </kitodo:metadata>
+                        </kitodo:kitodo>
+                    </mods:extension>
+                </mods:mods>
+            </mets:xmlData>
+        </mets:mdWrap>
+    </mets:dmdSec>
+    <mets:dmdSec ID="DMDLOG_0001">
+        <mets:mdWrap MDTYPE="MODS">
+            <mets:xmlData>
+                <mods:mods xmlns:mods="http://www.loc.gov/mods/v3">
+                    <mods:extension>
+                        <kitodo:kitodo xmlns:kitodo="http://meta.kitodo.org/v1/" version="1.0">
+                            <kitodo:metadata name="TitleDocMain">[Seite 134r-156v]</kitodo:metadata>
+                        </kitodo:kitodo>
+                    </mods:extension>
+                </mods:mods>
+            </mets:xmlData>
+        </mets:mdWrap>
+    </mets:dmdSec>
+    <mets:dmdSec ID="DMDLOG_0002">
+        <mets:mdWrap MDTYPE="MODS">
+            <mets:xmlData>
+                <mods:mods xmlns:mods="http://www.loc.gov/mods/v3">
+                    <mods:extension>
+                        <kitodo:kitodo xmlns:kitodo="http://meta.kitodo.org/v1/" version="1.0">
+                            <kitodo:metadata name="TitleDocMain">[Seite 157r-181v]</kitodo:metadata>
+                        </kitodo:kitodo>
+                    </mods:extension>
+                </mods:mods>
+            </mets:xmlData>
+        </mets:mdWrap>
+    </mets:dmdSec>
+    <mets:fileSec>
+        <mets:fileGrp USE="LOCAL">
+            <mets:file ID="FILE_0001" MIMETYPE="image/tiff">
+                <mets:FLocat xmlns:xlink="http://www.w3.org/1999/xlink" LOCTYPE="URL" xlink:href="file:/opt/goobi/production/metadata/231263/images/AbulOtsa_PPN866273492_tif/00000001.tif"/>
```

Comment:
Is it possible to change path information from `/opt/goobi/production/metadata` to `/usr/local/kitodo/metadata`?

Reply:
done